Test/ModalWindow.test.ts

```diff
@@ -0,0 +1,25 @@
+import modalWindow from '../Src/UIComponents/ModalWindow/ModalWindow';
+
+describe('Modal Window test', () => {
+    test('return HTMLElement', () => {
+        const component = document.createElement('p');
```

Comment:
maybe component can be created outside of a function to do it just once instead of recreating it per each test case?

Reply:
Good idea 👍